app-frontend/src/app/components/pages/project/analyses/analyses/index.js

```diff
@@ -218,7 +219,7 @@ class AnalysesListController {
             .then(this.autoGenerateRenderDef.bind(this))
             .then(() => {
                 const tileUrl = this.analysisService.getAnalysisTileUrl(analysisId);
-                return L.tileLayer(tileUrl, { maxZoom: 30 });
+                return L.tileLayer(tileUrl, { maxZoom: BUILDCONFIG.TILES_MAX_ZOOM });
```

Comment:
use maxNativeZoom instead so it scales from the max zoom level to whatever the user is zoomed into.
With maxZoom, it doesn't fetch tiles at all past the specified zoom level

docs: https://leafletjs.com/reference-1.0.0.html#tilelayer-maxnativezoom